tidy3d/components/simulation.py

```diff
@@ -3638,10 +3640,12 @@ def snap_to_grid(geom: Geometry, axis: Axis) -> Geometry:
                 temp_geometry = set_bounds(snapped_geometry, bounds=new_bounds, axis=axis)
                 temp_structure = structure.updated_copy(geometry=temp_geometry, medium=new_medium)
 
-                if structure.medium.is_pec:
-                    pec_plus = increment_float(snapped_center, 1.0)
-                    pec_minus = increment_float(snapped_center, -1.0)
-                    new_bounds = (pec_minus, pec_plus)
+                # new_bounds = ((1.0+100*fp_eps)*snapped_center - dls[0] / 2, (1.0-100*fp_eps)*snapped_center + dls[1] / 2)
+                # if structure.medium.is_pec:
+                # pec_plus = increment_float(snapped_center, 1.0)
+                # pec_minus = increment_float(snapped_center, -1.0)
+                # new_bounds = (pec_minus, pec_plus)
+                new_bounds = (snapped_center, snapped_center)
```

Comment:
@dmarek-flex two questions:

- All 2d materials are treated as 0 thickness now?
- What's the difference does it make by turning off slight expansion?